src/runtime/bootstrap-lazy.ts

```diff
@@ -185,16 +185,20 @@ export const bootstrapLazy = (lazyBundles: d.LazyBundlesRuntimeData, options: d.
     });
   });
 
+  // These style should always be constructed and inserted into the DOM
+  // We'll conditionally add some more styles later on
+  dataStyles.innerHTML = SLOT_FB_CSS;
```

Comment:
Double checking - why should we always add this style to DOM, even if a component doesn't have any slots? I think that's still lost on me, even with the added comment

Reply:
We shouldn't. Didn't realize we know specifically if any components are using slots outside the Shadow DOM. So, I added a boolean that will be set `true` if any components in the lazy build do that and that is used to conditionally add the `slot-fb` styles to the stylesheet in the DOM head.